samples/charts/data-chart/custom-drawing-annotations/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "scripts": {
+    "ng": "ng",
+    "update": "ng update",
+    "start": "node --max_old_space_size=12192 node_modules/@angular/cli/bin/ng serve -o",
+    "build": "node --max_old_space_size=12192 node_modules/@angular/cli/bin/ng build --configuration production",
+    "lint": "ng lint"
+  },
+  "dependencies": {
+    "@angular/animations": "16.0.1",
+    "@angular/common": "16.0.1",
+    "@angular/compiler": "16.0.1",
+    "@angular/core": "16.0.1",
+    "@angular/forms": "16.0.1",
+    "@angular/platform-browser": "16.0.1",
+    "@angular/platform-browser-dynamic": "16.0.1",
+    "@types/hammerjs": "2.0.39",
+    "classlist.js": "1.1.20150312",
+    "core-js": "2.6.2",
+    "hammerjs": "2.0.8",
+	"igniteui-angular-layouts": "16.1.0",
+    "igniteui-angular-inputs": "16.1.0",
+    "igniteui-angular-charts": "16.1.0",
+    "igniteui-angular-core": "16.1.0",
```

Comment:
use proget packages, e.g. in all new samples
    "@infragistics/igniteui-angular-charts": "23.2.18",
because Mike is updating existing samples to use proget packages